init.zsh

```diff
@@ -7,8 +7,12 @@ if [[ -z "$ANTIGEN_HS_MY" ]]; then
     ANTIGEN_HS_MY="$ANTIGEN_HS_HOME/../MyAntigen.hs"
 fi
 
+antigen-hs-init () {
+  cd "$ANTIGEN_HS_HOME" && stack build
+}
+
 antigen-hs-compile () {
-  runghc -i"$ANTIGEN_HS_HOME/" -- "$ANTIGEN_HS_MY"
+  stack exec -- runghc -i"$ANTIGEN_HS_HOME/" -- "$ANTIGEN_HS_MY"
```

Comment:
I'm trying to make antigen-hs easy to install and get going with. Adding a dependency for people who've never used Haskell does not make sense to me. I'm fine with adding everything in this PR except for the parts that forces the users into doing the "right way" from a sandboxing perspective. It's just not practical.